.werft/jobs/build/prepare.ts

```diff
@@ -1,24 +1,52 @@
-import * as shell from 'shelljs';
 import { exec } from '../../util/shell';
 import { Werft } from "../../util/werft";
 import { GCLOUD_SERVICE_ACCOUNT_PATH } from "./const";
 
+const phaseName = "prepare";
+
 export async function prepare(werft: Werft) {
-    werft.phase("prepare");
+    werft.phase(phaseName);
+    try {
+        compareWerftAndGitpodImage()
+        activateCoreDevServiceAccount()
+        configureDocker()
+        configureCoreDevAccess()
+    } catch (err) {
+        werft.fail(phaseName, err);
+    }
+    werft.done(phaseName);
+}
 
-    const werftImg = shell.exec("cat .werft/build.yaml | grep dev-environment").trim().split(": ")[1];
-    const devImg = shell.exec("yq r .gitpod.yml image").trim();
+// TODO: The reasoning behind this step should be clarified
```

Comment:
it's because both images need to be updated separately and letting them go out of sync may cause weird follow-up errors.